src/js/worker/core/player.js

```diff
@@ -668,15 +671,25 @@ function genRatings(
     season: number,
     scoutingRank: number,
     tid: number,
+	predeterminedHeight: number,
 ): PlayerRatings {
     let profileId;
-    if (profile === "Point") {
+    if (predeterminedHeight !== undefined) {
+		if  ((predeterminedHeight > 55) && random.randInt(0,1000) < 999)
```

Comment:
Indentation is off. Also please always use brackets around blocks. Really, I should start using https://github.com/prettier/prettier